client/js/libs/handlebars/ircmessageparser/merge.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+const _ = require("lodash");
```

Comment:
Right, but I was pretty sure we dropped IE support a long time ago and we expected Windows users to run Edge. @xPaw, what do you think?
If not, then we should use https://babeljs.io/docs/plugins/transform-object-assign/ instead, no?